src/Network/Receive/Zero.pm

```diff
@@ -24,6 +24,13 @@ sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
 
+	my %packets = (
+		'0ADD' => ['item_appeared', 'a4 V v C v2 C2 v C v', [qw(ID nameID type identified x y subx suby amount show_effect effect_type)]],
+	);
+
+	$self->{packet_list}{$_} = $packets{$_} for keys %packets;
+=pod
+# I think this is extra code in the received packages
```

Comment:
this is used in xkore 3 and 2, if not set, openkore will use old character pages and other old packets, and cRo has support for both cause use nprotect gameguard